profiles/common/modules/features/ec_embedded_image/ec_embedded_image.behat.inc

```diff
@@ -86,34 +86,21 @@ class ECEmbeddedImageSubContext extends DrupalSubContextBase {
       ]
     );
 
-    $this->contentTypeWasSetUp = TRUE;
+    $this->testPermissionsSetUp = TRUE;
   }
 
   /**
-   * Removes the test_media_gallery content type and related permissions.
+   * Removes the permissions related to the test_media_gallery content type.
+   *
+   * @see createMediaGalleryContentType()
    *
    * @AfterScenario
    */
-  public function removeMediaGalleryContentType() {
-    if (!$this->contentTypeWasSetUp) {
+  public function resetMediaGalleryPermission() {
+    if (!$this->testPermissionsSetUp) {
       return;
     }
 
-    $this->contentTypeWasSetUp = FALSE;
```

Comment:
@dxvargas , 
I have a question.
When we run all scenarios, do we use the same class instance through all scenarios or a class instance is created for each scenario?
According to what I understood from web articles about the question, the answer is "an instance by scenario" and then, at each scenario, the value will be false (default value of the class attribute).

If I implement what you ask, Then, that means that according to you, we use the same instance through all scenarios? Could you please confirm that before I start to touch my code.